tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -1282,6 +1282,9 @@ static bool shouldIgnore(Decl *D) {
       return true;
     if (VD->getBaseName().empty())
       return true;
+    // This shouldn't happen, being forgiving here.
+    if (!VD->hasAccess())
+      return true;
```

Comment:
I suggest adding a VD->dump() call here and running your test, then work backwards from there

Reply:
I have limited bandwidth for now. I'm not merging this.

Reply:
I'm not opposed to merging this, I just think you or @jrose-apple should investigate the underlying problem eventually :)